### DISCUSSION TITLE
Sharing links to a playlist of pages

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [alvarlagerlof/neurodiversity#294] [Ideas]
Let's say you've got a friend/relative who is unaware of your conditions. Instead of sending the multiple pages in a row, what if you could send then a link with all of the right pages included? What if you could also add a comment for each page?

![image](https://user-images.githubusercontent.com/14835120/120405487-f3c88b00-c348-11eb-9e1f-2262f26f4af9.png)
![image](https://user-images.githubusercontent.com/14835120/120405537-0fcc2c80-c349-11eb-8aec-45c9e303edfe.png)

**Comment by Toasterbirb** [comment]
Maximum URL length seems to be 2000 characters. Shouldn't be too difficult to jam a couple of variables in there and a short comment

**Comment by alvarlagerlof** [comment]
Good point there. I did at first think about storing it in some kind of database and then referencing it with and ID, but that seems risky considering the sensitive nature.

**Comment by alvarlagerlof** [comment]
@Toasterbirb Moved here
